src/MainView.vala

```diff
@@ -28,6 +28,12 @@ public class DateTime.MainView : Gtk.Grid {
 
     private static GLib.Settings time_zone_settings;
 
+    public MainView () {
```

Comment:
I don't think we need to add a constructor for this. This value isn't passed as a construction argument. It can just go above the call to `add ()`